cg/services/order_validation_service/workflows/fastq/validation_service.py

```diff
@@ -21,14 +20,12 @@ def __init__(self, store: Store):
         self.store = store
 
     def validate(self, raw_order: dict) -> dict:
-        errors: ValidationErrors = self._get_errors(raw_order)
+        parsed_order, errors = ModelValidator.validate(order=raw_order, model=FastqOrder)
+        if parsed_order:
```

Comment:
So we only want to apply the rules when the pydantic validation has passed?

Reply:
The rules all take a parsed Pydantic BaseModel as input so there is no other option. It will be a bit weird for customers that they do not get all the errors at the same time, but without a massive rework I do not think we have a choice

Reply:
Like if they send a request which looks like `{"message": "Hello world!"}` there is not much validation that can be performed apart from stating that we can not parse it as an order